installer/Migration/V5_1_0/lang-string/general.yaml

```diff
@@ -173,6 +167,3 @@ langStrings:
   -
```

Comment:
remove this note from 5.1 since already there. I checked in 5.0

Reply:
Will remove it

Reply:
Done